src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -0,0 +1,68 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines Functions for calculating spacetime tensors from 3+1 quantities
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace GeneralizedHarmonic {
+/*!
+ * \ingroup GeneralRelativityGroup
```

Comment:
I'm not sure if we want them to be in this group. There's no GH group (and I don't think there should be), so I'm actually in favor of having the GH functions also be in the GR group.